### ISSUE TITLE
Create standalone Tooltip component

### ISSUE THREAD

**Issue by dpikt** [LaunchPadLab/lp-components#289]

**Comment by chawes13** [comment]
@dpikt Thoughts on leveraging the accessible `<ToggleTip />` created for SC?

https://github.com/LaunchPadLab/streamlined-checkout-client/blob/dev/src/js/components/ToggleTip.js

**Comment by dpikt** [comment]
Looks like a great start! 
It would also be nice to have the option for external control.

**Comment by chawes13** [comment]
Similar to `Modal.show()`?

**Comment by dpikt** [comment]
Via props like `shown` and `onClick`. This would make it controllable: https://medium.com/myheritage-engineering/how-controllable-react-components-maximize-reusability-86e3d233fa8e

(Most of our components are controlled btw)

**Comment by dpikt** [comment]
Hmm, are there any important use cases for tooltips that toggletips don't cover? Might be good to just stick with this- I think the tooltips on our inputs toggle already.

**Comment by chawes13** [comment]
On second thought, we should probably use a library for this. Don't think it's worth rolling our own to handle older browsers and collision detection

**Comment by dpikt** [comment]
I'll defer to your judgement!

**Comment by dpikt** [comment]
Following up- Tippy seems like a pretty solid library to me. Thoughts on swapping out our default tooltip with it?

**Comment by dpikt** [comment]
Official react bindings for Tippy: https://github.com/atomiks/tippy.js-react

**Comment by ghrlee** [comment]
@chawes13, can I grab this?

**Comment by chawes13** [comment]
Sure! Do you feel like you have a clear path forward?

**Comment by ghrlee** [comment]
> 

Yeah, thanks, I'm looking into Tippy and it seems like the right package for this, but want to make sure that still sounds right and that it wasn't the reason this stalled before.

**Comment by chawes13** [comment]
@ghrlee I think it's worth investigating the react popper bindings (https://popper.js.org/react-popper/v2/) and comparing the DX
